Makefile

```
@@ -327,6 +327,12 @@ lint:  ${GOLANGCI_LINT} ## Run the golang linter, see also: .github/workflows/ve
 	${GOLANGCI_LINT} linters
 	GL_DEBUG=gocritic ${GOLANGCI_LINT} run
 
+.PHONY: lint-fix
+lint:  ${GOLANGCI_LINT} ## Run the golang linter and fix the issues
```

```suggestion
lint-fix:  ${GOLANGCI_LINT} ## Run the golang linter and fix the issues
```

Or we just run the litter always using `--fix` and diff afterwards?

---

Thanks, that's careless of me.

> Or we just run the litter always using --fix and diff afterwards?

That sounds better. https://github.com/golangci/golangci-lint/issues/3230 this kind of issue might happen, but let's see.

---

Sounds good!